embedded-kafka/src/main/scala/io/github/embeddedkafka/ops/kafkaOps.scala

```diff
@@ -1,62 +1,137 @@
 package io.github.embeddedkafka.ops
 
-import java.nio.file.Path
-import kafka.server.{KafkaConfig, KafkaServer}
-import io.github.embeddedkafka.{
-  EmbeddedK,
-  EmbeddedKafkaConfig,
-  EmbeddedServer,
-  EmbeddedZ
-}
+import io.github.embeddedkafka.{EmbeddedK, EmbeddedKafkaConfig, EmbeddedServer}
+import kafka.server._
 import org.apache.kafka.common.security.auth.SecurityProtocol
+import org.apache.kafka.common.utils.Time
 import org.apache.kafka.coordinator.group.GroupCoordinatorConfig
-import org.apache.kafka.coordinator.transaction.TransactionLogConfigs
+import org.apache.kafka.metadata.properties.{
+  MetaProperties,
+  MetaPropertiesEnsemble,
+  MetaPropertiesVersion,
+  PropertiesUtils
+}
 import org.apache.kafka.network.SocketServerConfigs
+import org.apache.kafka.raft.QuorumConfig
+import org.apache.kafka.server.ServerSocketFactory
 import org.apache.kafka.server.config.{
+  KRaftConfigs,
   ServerConfigs,
-  ServerLogConfigs,
-  ZkConfigs
+  ServerLogConfigs
 }
 import org.apache.kafka.storage.internals.log.CleanerConfig
+import org.slf4j.LoggerFactory
 
+import java.io.File
+import java.nio.file.{Path, Paths}
+import java.util.concurrent.CompletableFuture
 import scala.jdk.CollectionConverters._
 
 /**
   * Trait for Kafka-related actions.
   */
 trait KafkaOps {
-  protected val brokerId: Short                 = 0
+
+  private val logger = LoggerFactory.getLogger(getClass)
+
+  protected val nodeId: Int                     = 0
+  protected val clusterIdBase64                 = "H3KKO4NTRPaCWtEmm3vW7A"
```

Comment:
@gaeljw check if we can avoid the hardcoded value, either generate a random one or maybe let the default value?